src/style.scss

```diff
@@ -1,227 +1,55 @@
-$desktop-size: 960px;
-$tablet-size: 760px;
-$mobile-size: 320px;
-$has-border: false;
-// $has-border: true;
+@import url(//fonts.googleapis.com/css?family=Open+Sans);
 
 body {
-  background: black;
+  font-family: "Open Sans" !important;
 }
-
-.story {
-  display: flex;
-  margin: 2rem auto;
-  @if $has-border {
-    outline: 5px solid yellow;
-  }
-  max-width: calc(100vh * 16 / 9);
-  @media only screen and (min-width: $desktop-size) {
-    height: calc(100vh - 4rem);
-  }
-  @media only screen and (max-width: $desktop-size - 1) {
-    flex-direction: column;
-  }
-  &--wrap {
-    flex-wrap: wrap;
-    align-content: center;
-  }
+a,
+a:hover {
+  color: inherit;
+  text-decoration: none;
 }
-
-.content {
-  display: flex;
-  flex-basis: calc(100% / 3);
-  @if $has-border {
-    outline: 5px solid green;
-  }
-  &--x-center {
-    justify-content: center;
-  }
-  // 高度50% 所以不需要了
-  // &--x-spaceAround {
-  //   justify-content: space-around;
-  // }
-  &--vertical {
-    flex-direction: column;
-    align-items: center;
-  }
-  &--mobile-mb-3 {
-    @media only screen and (max-width: $desktop-size) {
-      margin-bottom: 3rem;
-    }
-  }
+.bgBlack {
+  background-color: #0e1414;
```

Comment:
Make use of bootstrap is a better approach. 
https://getbootstrap.com/docs/4.0/utilities/colors/#background-color

Reply:
`#0e1414` is from Invision, in Bootstrap we get dark one with `#343a40`
That's why I customize it.

Reply:
Is it easy to customise with bootstrap variables? If so, don't create your own one. https://getbootstrap.com/docs/4.0/getting-started/theming/#variable-defaults

Reply:
@Joseph I make many commit for convenience of review. 
After approve I would happy to squash them into one commit.